### ISSUE TITLE
MP3 Compression Settings Leave Hiss on Output

### ISSUE THREAD

**Issue by andspenrob** [audiamus/AaxAudioConverter#184]
I've been exporting from Audible with BookLibConnect and attempting to convert with AAX Audio Converter using default settings. The resulting MP3 files have a constant hiss that is not present in the *.m4b files generated during the BookLibConnect export. Both files were tested using VLC on Windows 11.

**Comment by audiamus** [comment]
AAX Audio Converter delegates all audio processing [FFmpeg](https://www.ffmpeg.org/). Using the defaults in AAC Audio Converter will also use the defaults of FFmpeg. Did the book in question have unusual bit and sample rates?

**Comment by andspenrob** [comment]
These are coming straight from Audible - most are 64kbps 22khz stereo (or so says BookLibConnect). I'm wondering if AAC defaults are trying too hard to re-compress? Hank Green's An Absolutely Remarkable Thing starts at 257MB in M4B with no hiss or clipping and actually ends up larger at 259MB in MP3. Is it going from VBR to CBR or something?

**Comment by andspenrob** [comment]
Hey, so I checked out the input (M4B) and output (MP3) files, and I found that the Audible M4B is, in fact, VBR at 64 kbps thru AAC-LC, while the MP3 output is Joint Stereo CBR at 64kbps. That explains the inflated file size, at least!

Now, it turns out my audiobook player can read M4B, so I don't personally need the second re-compression to MP3 or the auto-folders, but I thought you might want to know about the discrepancy. Not sure what best practice is, since recompression is lossy - maybe compress in VBR with more headroom like 96kbps for 64kbps files? Thanks for working thru it with me!
